pgscm/templates/admin/user.html

```diff
@@ -11,9 +12,26 @@
     <!-- DataTables with select2 enabled -->
     {{ load_datatables_script(
             ajax_endpoint='user',
-            export_columns='0,1,2,3,4,5,6',
+            export_columns='0,1,2,3,4',
             column_names=["associate_group_code", "name", "email", "province", "province"],
-            select2_class=g.c.SELECT_DEFAULT_ID) }}
+            select2_class=g.c.SELECT_DEFAULT_ID, multi_select2_class=g.c.MULTI_SELECT_DEFAULT_CLASS) }}
+    <script>
+        $('.{{ g.c.BTNEDIT_ID }}').on('click', function (event) {
```

Comment:
I know. I will also try to put js for select2 multiple to adminlte/init.py when i custom the default values for select2 multiple in Saturday.
But with code for special fields which only have in user form like old/new pass, confirm pass, i think we don't need to put code handle there field in the general.